pyaedt/filtersolutions_core/transmission_zeros.py

```diff
@@ -0,0 +1,258 @@
+
+        Returns
+        -------
+        tuple: The tuple contains
+                str:
+                    Transmission zero ratio or frequency value.
+                str:
+                    Position of the element creating transmission zero in the associated circuit.
+        """
+        zero_value_buffer = create_string_buffer(100)
+        position_value_buffer = create_string_buffer(100)
+        status = self._dll.getTransmissionZerosTableRow(
+            row_index,
+            zero_value_buffer,
+            position_value_buffer,
+            self.table_format_to_bool(),
+            100,
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        zero_value_string = zero_value_buffer.value.decode("utf-8")
+        position_value_string = position_value_buffer.value.decode("utf-8")
+        return zero_value_string, position_value_string
+
+    def update_row(self, row_index, zero="", position=""):
+        """Update ratios or frequencies at given index of the transmission zeros table.
+
+        Parameters
+        ----------
+        row_index: int
+            The row index on transmission zeros table. Starting value is ``0`` and maximum value is ``9``.
+        zero: str, optional
+            Transmission zero ratio or frequency value.
+            The default is blank.
+        position: str, optional
+            Position of the element creating transmission zero in the associated circuit.
+            The default is blank.
+        """
+        zero_bytes_value = bytes(zero, "ascii")
+        position_bytes_value = bytes(position, "ascii")
+        status = self._dll.updateTransmissionZerosTableRow(
+            row_index,
+            zero_bytes_value,
+            position_bytes_value,
+            self.table_format_to_bool(),
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def append_row(self, zero, position=""):
+        """Append ratios or frequencies at the end row of transmission zeros table.
+
+        Parameters
+        ----------
+        zero: str
+            Transmission zero ratio or frequency value.
+        position: str
+            Position of the element creating transmission zero in the associated circuit.
+        """
+        zero_bytes_value = bytes(zero, "ascii")
+        position_bytes_value = bytes(position, "ascii")
+        status = self._dll.appendTransmissionZerosTableRow(
+            zero_bytes_value, position_bytes_value, self.table_format_to_bool()
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def insert_row(self, row_index, zero, position=""):
+        """Insert ratios or frequencies at given index of the transmission zeros table.
+
+        Parameters
+        ----------
+        row_index: int
+            The row index on transmission zeros table. Starting value is ``0`` and maximum value is ``9``.
```

Comment:
What will happen to the existing row? Insert usually moves the row and all following rows down (up in index) by one.

Reply:
Will be fixed as a defect to move rows down the give index.